### ISSUE TITLE
Reflow doesn't apply to entire page, only paragraphs

### ISSUE THREAD

**Issue by KSSea** [rgh36167/FfxAndroidTextReflow#1]
When trying to reflow a page by double-tapping the reflow operation only applies to one paragraph.   Since most pages tend to have a lot of short paragraphs this means you have to keep double tapping over and over to actually read a page.

**Comment by rgh36167** [comment]
Yes, that is described in the description. I may improve that but it will never be fully automatic.

For the fully automatic version see

https://addons.mozilla.org/en-US/android/addon/fit-text-to-width/?src=ss
